easybuild/easyconfigs/f/fastStructure/fastStructure-1.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,24 @@
+name = 'fastStructure'
+version = '1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://rajanil.github.io/fastStructure/'
+description = """
+fastStructure is an algorithm for inferring population structure 
+from large SNP genotype data. It is based on a variational Bayesian 
+framework for posterior inference and is written in Python2.x. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/rajanil/fastStructure/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f1bfb24bb5ecd108bc3a90145fad232012165c1e60608003f1c87d200f867b81']
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('matplotlib', '1.5.1', '-Python-%(pyver)s'),
```

Comment:
`1.5.1` for this toolchain is not in the repository, is this exact version needed? if not, there are easyconfigs for `1.5.2`, `1.5.3` and `2.0.0`